pytest.ini

```diff
@@ -16,5 +16,5 @@ env =
     ARKOUDA_SERVER_HOST=localhost
     ARKOUDA_SERVER_PORT=5555
     ARKOUDA_FULL_STACK_TEST=True
-    ARKOUDA_NUMLOCALES=2
+    D:ARKOUDA_NUMLOCALES=2
     ARKOUDA_VERBOSE=False
```

Comment:
Would you be up for making all these env vars `D:` prefixed? For better or worse (probably worse) HPC people tend to be env var oriented, so if I was going to do a client session, I'd probably do so with the env vars instead of modifying the `.ini` file

Reply:
Indeed! Just made the change and pushed to git